includes/amp-helper-functions.php

```diff
@@ -929,9 +929,20 @@ function amp_get_content_sanitizers( $post = null ) {
 	}
 
 	if ( $amp_to_amp_linking_enabled ) {
-		$sanitizers['AMP_Link_Sanitizer'] = [
-			'paired' => ! amp_is_canonical(),
-		];
+
+		/**
+		 * Normally, when in a paired mode, links to the same origin will be to AMP.
+		 *
+		 * This allows passing URLs to exclude from having AMP-to-AMP links.
+		 *
```

Comment:
Please add a `@since` version here.

Reply:
Sure, it's added in https://github.com/ampproject/amp-wp/pull/4146/commits/160eb877bb8b838ddc61a60b5c60e9c2e93ee0ff